redis/server/install.sls

```diff
@@ -0,0 +1,104 @@
+include:
+  - redis.common
+
+{% from "redis/map.jinja" import redis_settings as r with context %}
+
+    {%- if r.install_from == 'source' %}
+
+redis_user_group:
+  group.present:
+    - name: {{ r.group }}g
+  user.present:
+    - name: {{ r.user }}g
```

Comment:
It's hard to understand why the final `g`. It looks like a typo.

If the final `g` is mandatory, it may be better directly in `defaults.yaml`.

Reply:
Definitely a typo!!!